nvflare/private/fed/server/server_engine_internal_spec.py

```diff
@@ -195,6 +197,16 @@ def set_run_manager(self, run_manager: RunManager):
         """
         pass
 
+    @abstractmethod
+    def set_job_runner(self, job_runner: JobRunner, job_manager: JobDefManagerSpec):
```

Comment:
since it set both job_runner and job_manager,
should we change the name to reflects this?